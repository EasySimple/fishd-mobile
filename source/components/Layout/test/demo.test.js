import webDemoTest from '../../../tests/demoTest';

webDemoTest('Layout');
