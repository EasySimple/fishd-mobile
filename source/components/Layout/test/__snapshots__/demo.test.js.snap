// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Layout demo correctly 1`] = `
<div>
  <div
    class="components-layout-demo-basic"
  >
    <div
      class="demo-title"
    >
      基础用法
    </div>
    <div
      class="fm-row"
    >
      <div
        class="fm-col fm-col-12"
      >
        ospan:12
      </div>
      <div
        class="fm-col fm-col-12"
      >
        span:12
      </div>
    </div>
    <div
      class="fm-row"
    >
      <div
        class="fm-col fm-col-8"
      >
        span:8
      </div>
      <div
        class="fm-col fm-col-8"
      >
        span:8
      </div>
      <div
        class="fm-col fm-col-8"
      >
        span:8
      </div>
    </div>
    <div
      class="fm-row"
    >
      <div
        class="fm-col fm-col-8"
      >
        span:8
      </div>
      <div
        class="fm-col fm-col-8"
      >
        span:8
      </div>
      <div
        class="fm-col fm-col-8"
      >
        span:8
      </div>
    </div>
    <div
      class="fm-row"
    >
      <div
        class="fm-col fm-col-4"
      >
        span:4
      </div>
      <div
        class="fm-col fm-col-10 fm-col-offset-4"
      >
        offset:4,span:10
      </div>
    </div>
    <div
      class="demo-title"
    >
      元素间增加间距
    </div>
    <div
      class="fm-row"
    >
      <div
        class="fm-col fm-col-12"
        style="padding-left:0px;padding-right:10px"
      >
        span: 12
      </div>
      <div
        class="fm-col fm-col-12"
        style="padding-left:10px;padding-right:0px"
      >
        span: 12
      </div>
    </div>
    <div
      class="fm-row"
    >
      <div
        class="fm-col fm-col-8"
        style="padding-left:0px;padding-right:13.333333333333334px"
      >
        span: 8
      </div>
      <div
        class="fm-col fm-col-8"
        style="padding-left:6.666666666666666px;padding-right:6.666666666666668px"
      >
        span: 8
      </div>
      <div
        class="fm-col fm-col-8"
        style="padding-left:13.333333333333332px;padding-right:1.7763568394002505e-15px"
      >
        span: 8
      </div>
    </div>
    <div
      class="fm-row"
    >
      <div
        class="fm-col fm-col-6"
        style="padding-left:0px;padding-right:15px"
      >
        span: 6
      </div>
      <div
        class="fm-col fm-col-6"
        style="padding-left:5px;padding-right:10px"
      >
        span: 6
      </div>
      <div
        class="fm-col fm-col-6"
        style="padding-left:10px;padding-right:5px"
      >
        span: 6
      </div>
      <div
        class="fm-col fm-col-6"
        style="padding-left:15px;padding-right:0px"
      >
        span: 6
      </div>
    </div>
    <div
      class="demo-title"
    >
      柔性布局
    </div>
    <div
      class="fm-row fm-row-flex fm-row-align-top fm-row-justify-start"
    >
      <div
        class="fm-col fm-col-6"
      >
        span: 6
      </div>
      <div
        class="fm-col fm-col-6"
      >
        span: 6
      </div>
      <div
        class="fm-col fm-col-6"
      >
        span: 6
      </div>
    </div>
    <div
      class="fm-row fm-row-flex fm-row-align-top fm-row-justify-center"
    >
      <div
        class="fm-col fm-col-6"
      >
        span: 6
      </div>
      <div
        class="fm-col fm-col-6"
      >
        span: 6
      </div>
      <div
        class="fm-col fm-col-6"
      >
        span: 6
      </div>
    </div>
    <div
      class="fm-row fm-row-flex fm-row-align-top fm-row-justify-end"
    >
      <div
        class="fm-col fm-col-6"
      >
        span: 6
      </div>
      <div
        class="fm-col fm-col-6"
      >
        span: 6
      </div>
      <div
        class="fm-col fm-col-6"
      >
        span: 6
      </div>
    </div>
    <div
      class="fm-row fm-row-flex fm-row-align-top fm-row-justify-space-between"
    >
      <div
        class="fm-col fm-col-6"
      >
        span: 6
      </div>
      <div
        class="fm-col fm-col-6"
      >
        span: 6
      </div>
      <div
        class="fm-col fm-col-6"
      >
        span: 6
      </div>
    </div>
    <div
      class="fm-row fm-row-flex fm-row-align-top fm-row-justify-space-around"
    >
      <div
        class="fm-col fm-col-6"
      >
        span: 6
      </div>
      <div
        class="fm-col fm-col-6"
      >
        span: 6
      </div>
      <div
        class="fm-col fm-col-6"
      >
        span: 6
      </div>
    </div>
  </div>
</div>
`;
